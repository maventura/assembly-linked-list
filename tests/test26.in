--- test 0 de 50
crear: tipo 2. valor 0c2i
crear: tipo 2. valor 80i037bn
concatenar: nodo 0
crear: tipo 2. valor y6lios
crear: tipo 2. valor 8qaajik6i
concatenar: nodo 2
crear: tipo 2. valor nxefwlb
imprimir: antes   de filtrar:
filtrar: tipo 2
imprimir: despues de filtrar:
ahijar: nodo 3 en columna 42
concatenar: nodo 1
crear: tipo 2. valor c9y5lcq
crear: tipo 2. valor f7l8ucyi
crear: tipo 2. valor 55c5
crear: tipo 2. valor f8hhj
crear: tipo 2. valor l
concatenar: nodo 6
crear: tipo 2. valor 77849
ahijar: nodo 9 en columna 1
crear: tipo 2. valor x0vewc
concatenar: nodo 10
concatenar: nodo 8
concatenar: nodo 5
crear: tipo 2. valor 4k
crear: tipo 2. valor 3
crear: tipo 2. valor xhctx0jx
ahijar: nodo 11 en columna 7
concatenar: nodo 12
concatenar: nodo 4
concatenar: nodo 7
concatenar: nodo 13
crear: tipo 2. valor am3p9v
crear: tipo 2. valor 31lm1
crear: tipo 2. valor 5gswp8yx5w
crear: tipo 2. valor c
crear: tipo 2. valor s
ahijar: nodo 19 en columna 2
ahijar: nodo 15 en columna 23
concatenar: nodo 18
imprimir: antes   de filtrar:
filtrar: tipo 2
imprimir: despues de filtrar:
crear: tipo 2. valor dpav9ao7b
imprimir: antes   de colapsar
colapsar: tipo 2
imprimir: despues de colapsar
crear: tipo 2. valor 4ry6
crear: tipo 2. valor ecv
concatenar: nodo 21
crear: tipo 2. valor p0jjf0
crear: tipo 2. valor vs8auox
crear: tipo 2. valor xrd2v
concatenar: nodo 25
concatenar: nodo 14
concatenar: nodo 16
concatenar: nodo 17
crear: tipo 2. valor ry
crear: tipo 2. valor nean0o
crear: tipo 2. valor drqswomm
crear: tipo 2. valor o39
concatenar: nodo 26
concatenar: nodo 23
crear: tipo 2. valor s6asjc
ahijar: nodo 27 en columna 48
ahijar: nodo 29 en columna 0
crear: tipo 2. valor ek0vc4ah
imprimir: antes   de colapsar
colapsar: tipo 2
imprimir: despues de colapsar
ahijar: nodo 22 en columna 47
imprimir: antes   de filtrar:
filtrar: tipo 2
imprimir: despues de filtrar:
crear: tipo 2. valor puky7f8l
crear: tipo 2. valor ank
concatenar: nodo 24
imprimir: antes   de colapsar
colapsar: tipo 2
imprimir: despues de colapsar
crear: tipo 2. valor mebsfux
crear: tipo 2. valor 9
crear: tipo 2. valor mcr4m3vxn8
crear: tipo 2. valor n
crear: tipo 2. valor ax
concatenar: nodo 20
crear: tipo 2. valor 1he064ju
concatenar: nodo 35
concatenar: nodo 33
crear: tipo 2. valor wlkwxajsmv
crear: tipo 2. valor axb
crear: tipo 2. valor q
crear: tipo 2. valor yige
crear: tipo 2. valor d1fon31a
crear: tipo 2. valor hrdirel
crear: tipo 2. valor r96ghoxch
imprimir: antes   de colapsar
colapsar: tipo 2
imprimir: despues de colapsar
concatenar: nodo 38
imprimir: antes   de colapsar
colapsar: tipo 2
imprimir: despues de colapsar
crear: tipo 2. valor y3ay
imprimir: antes   de colapsar
colapsar: tipo 2
imprimir: despues de colapsar
concatenar: nodo 46
concatenar: nodo 34
imprimir: antes   de colapsar
colapsar: tipo 2
imprimir: despues de colapsar
ahijar: nodo 41 en columna 22
crear: tipo 2. valor fgd9f9r01
crear: tipo 2. valor xvil5j
ahijar: nodo 44 en columna 14
ahijar: nodo 37 en columna 3
concatenar: nodo 42
ahijar: nodo 40 en columna 40
concatenar: nodo 39
concatenar: nodo 36
concatenar: nodo 30
ahijar: nodo 31 en columna 49
ahijar: nodo 28 en columna 43
concatenar: nodo 47
concatenar: nodo 45
concatenar: nodo 48
concatenar: nodo 49
concatenar: nodo 43
ahijar: nodo 32 en columna 32
borrar
--- test 1 de 50
crear: tipo 2. valor uvyuo6p
crear: tipo 2. valor so7x
concatenar: nodo 0
concatenar: nodo 1
crear: tipo 2. valor mwd869
crear: tipo 2. valor 29
concatenar: nodo 2
concatenar: nodo 3
crear: tipo 2. valor mu3d4qjl
crear: tipo 2. valor 24dojn
crear: tipo 2. valor bl2
ahijar: nodo 6 en columna 41
crear: tipo 2. valor k
imprimir: antes   de colapsar
colapsar: tipo 2
imprimir: despues de colapsar
imprimir: antes   de filtrar:
filtrar: tipo 2
imprimir: despues de filtrar:
crear: tipo 2. valor i
crear: tipo 2. valor ma4ldvssa2
crear: tipo 2. valor 5yyw
ahijar: nodo 10 en columna 8
concatenar: nodo 8
ahijar: nodo 9 en columna 13
ahijar: nodo 5 en columna 12
concatenar: nodo 7
ahijar: nodo 4 en columna 28
crear: tipo 2. valor osq1
imprimir: antes   de colapsar
colapsar: tipo 2
imprimir: despues de colapsar
crear: tipo 2. valor x
crear: tipo 2. valor t
concatenar: nodo 12
concatenar: nodo 13
crear: tipo 2. valor csulrdk4w
crear: tipo 2. valor 4
crear: tipo 2. valor 9daaa10u
ahijar: nodo 11 en columna 45
crear: tipo 2. valor 48u1kc5
crear: tipo 2. valor nc1tofrovr
crear: tipo 2. valor q
crear: tipo 2. valor 7ih
crear: tipo 2. valor sfe8mc4g
crear: tipo 2. valor 9xp3onwfm
ahijar: nodo 21 en columna 20
crear: tipo 2. valor 8fdywv5e6t
concatenar: nodo 15
concatenar: nodo 18
crear: tipo 2. valor a3p2t
ahijar: nodo 16 en columna 25
ahijar: nodo 20 en columna 46
concatenar: nodo 24
crear: tipo 2. valor xbwvi1s2j
crear: tipo 2. valor 8ugkovo8ab
concatenar: nodo 14
concatenar: nodo 19
crear: tipo 2. valor m240o
ahijar: nodo 27 en columna 19
crear: tipo 2. valor 7km565rw
crear: tipo 2. valor 3kd
concatenar: nodo 25
crear: tipo 2. valor 6s20is
crear: tipo 2. valor rw4oj
crear: tipo 2. valor nqeg0b
crear: tipo 2. valor q6g8i
ahijar: nodo 29 en columna 12
crear: tipo 2. valor 98u371ugj
crear: tipo 2. valor 93ryh
concatenar: nodo 22
concatenar: nodo 26
concatenar: nodo 28
imprimir: antes   de colapsar
colapsar: tipo 2
imprimir: despues de colapsar
crear: tipo 2. valor br
crear: tipo 2. valor a0k39nng9
ahijar: nodo 36 en columna 26
concatenar: nodo 32
crear: tipo 2. valor u9u57
crear: tipo 2. valor l3jav
concatenar: nodo 31
concatenar: nodo 39
crear: tipo 2. valor 6gk30ibca9
concatenar: nodo 35
crear: tipo 2. valor i8t
crear: tipo 2. valor ivxt8l
crear: tipo 2. valor 3mrdfhlgkx
imprimir: antes   de filtrar:
filtrar: tipo 2
imprimir: despues de filtrar:
crear: tipo 2. valor rr00pwy
crear: tipo 2. valor amglxy
concatenar: nodo 23
concatenar: nodo 41
ahijar: nodo 17 en columna 40
concatenar: nodo 45
crear: tipo 2. valor w
imprimir: antes   de colapsar
colapsar: tipo 2
imprimir: despues de colapsar
crear: tipo 2. valor 2kgt768vs9
crear: tipo 2. valor 9n1lih3g
concatenar: nodo 43
concatenar: nodo 47
concatenar: nodo 48
concatenar: nodo 37
imprimir: antes   de filtrar:
filtrar: tipo 2
imprimir: despues de filtrar:
crear: tipo 2. valor k7
concatenar: nodo 40
concatenar: nodo 49
imprimir: antes   de colapsar
colapsar: tipo 2
imprimir: despues de colapsar
concatenar: nodo 46
imprimir: antes   de colapsar
colapsar: tipo 2
imprimir: despues de colapsar
concatenar: nodo 38
concatenar: nodo 34
concatenar: nodo 33
ahijar: nodo 42 en columna 47
concatenar: nodo 30
imprimir: antes   de colapsar
colapsar: tipo 2
imprimir: despues de colapsar
ahijar: nodo 44 en columna 23
borrar
--- test 2 de 50
imprimir: antes   de filtrar:
filtrar: tipo 0
imprimir: despues de filtrar:
imprimir: antes   de colapsar
colapsar: tipo 0
imprimir: despues de colapsar
crear: tipo 0. valor 2581
ahijar: nodo 0 en columna 23
crear: tipo 0. valor -1140
crear: tipo 0. valor 3019
crear: tipo 0. valor -4041
crear: tipo 0. valor 1699
concatenar: nodo 3
ahijar: nodo 4 en columna 24
crear: tipo 0. valor -2571
crear: tipo 0. valor -3350
concatenar: nodo 6
crear: tipo 0. valor 1060
concatenar: nodo 5
concatenar: nodo 7
crear: tipo 0. valor -3309
concatenar: nodo 1
concatenar: nodo 2
concatenar: nodo 8
crear: tipo 0. valor -4062
crear: tipo 0. valor 311
concatenar: nodo 10
concatenar: nodo 9
crear: tipo 0. valor -2457
concatenar: nodo 11
crear: tipo 0. valor 2282
crear: tipo 0. valor 1428
ahijar: nodo 12 en columna 40
crear: tipo 0. valor -3874
crear: tipo 0. valor -516
crear: tipo 0. valor -921
concatenar: nodo 13
concatenar: nodo 16
ahijar: nodo 14 en columna 42
crear: tipo 0. valor 2120
concatenar: nodo 17
crear: tipo 0. valor 2830
crear: tipo 0. valor 22
crear: tipo 0. valor -3511
ahijar: nodo 20 en columna 21
concatenar: nodo 15
imprimir: antes   de colapsar
colapsar: tipo 0
imprimir: despues de colapsar
imprimir: antes   de colapsar
colapsar: tipo 0
imprimir: despues de colapsar
ahijar: nodo 18 en columna 17
concatenar: nodo 19
crear: tipo 0. valor 3696
concatenar: nodo 21
crear: tipo 0. valor -3233
crear: tipo 0. valor -4863
crear: tipo 0. valor -2014
ahijar: nodo 23 en columna 7
crear: tipo 0. valor -4353
concatenar: nodo 22
ahijar: nodo 25 en columna 41
concatenar: nodo 24
crear: tipo 0. valor 1830
crear: tipo 0. valor 3667
concatenar: nodo 26
crear: tipo 0. valor -4449
crear: tipo 0. valor 2694
concatenar: nodo 29
concatenar: nodo 27
crear: tipo 0. valor -4475
concatenar: nodo 30
ahijar: nodo 28 en columna 49
crear: tipo 0. valor -353
concatenar: nodo 31
crear: tipo 0. valor -3382
ahijar: nodo 32 en columna 29
crear: tipo 0. valor -2505
crear: tipo 0. valor 2321
crear: tipo 0. valor 551
imprimir: antes   de colapsar
colapsar: tipo 0
imprimir: despues de colapsar
ahijar: nodo 34 en columna 4
crear: tipo 0. valor 2194
concatenar: nodo 36
concatenar: nodo 33
imprimir: antes   de colapsar
colapsar: tipo 0
imprimir: despues de colapsar
crear: tipo 0. valor -3448
concatenar: nodo 37
crear: tipo 0. valor 4894
crear: tipo 0. valor -910
ahijar: nodo 35 en columna 22
concatenar: nodo 38
imprimir: antes   de filtrar:
filtrar: tipo 0
imprimir: despues de filtrar:
ahijar: nodo 39 en columna 15
crear: tipo 0. valor 3457
concatenar: nodo 40
crear: tipo 0. valor -969
imprimir: antes   de colapsar
colapsar: tipo 0
imprimir: despues de colapsar
concatenar: nodo 41
crear: tipo 0. valor -3743
crear: tipo 0. valor -461
concatenar: nodo 43
imprimir: antes   de filtrar:
filtrar: tipo 0
imprimir: despues de filtrar:
ahijar: nodo 42 en columna 48
imprimir: antes   de colapsar
colapsar: tipo 0
imprimir: despues de colapsar
crear: tipo 0. valor -4732
crear: tipo 0. valor -488
crear: tipo 0. valor 1160
concatenar: nodo 44
crear: tipo 0. valor 764
crear: tipo 0. valor -1433
crear: tipo 0. valor 555
concatenar: nodo 45
concatenar: nodo 48
concatenar: nodo 46
concatenar: nodo 47
ahijar: nodo 49 en columna 48
borrar
--- test 3 de 50
crear: tipo 0. valor 4894
ahijar: nodo 0 en columna 6
crear: tipo 0. valor 3295
crear: tipo 0. valor 3509
imprimir: antes   de filtrar:
filtrar: tipo 0
imprimir: despues de filtrar:
crear: tipo 0. valor 1054
ahijar: nodo 1 en columna 46
crear: tipo 0. valor 4034
crear: tipo 0. valor 176
concatenar: nodo 3
imprimir: antes   de colapsar
colapsar: tipo 0
imprimir: despues de colapsar
imprimir: antes   de colapsar
colapsar: tipo 0
imprimir: despues de colapsar
crear: tipo 0. valor 1268
ahijar: nodo 2 en columna 36
concatenar: nodo 6
concatenar: nodo 5
crear: tipo 0. valor -1073
crear: tipo 0. valor 2078
imprimir: antes   de filtrar:
filtrar: tipo 0
imprimir: despues de filtrar:
crear: tipo 0. valor 2032
crear: tipo 0. valor 3616
imprimir: antes   de colapsar
colapsar: tipo 0
imprimir: despues de colapsar
concatenar: nodo 10
concatenar: nodo 9
concatenar: nodo 4
crear: tipo 0. valor -2325
concatenar: nodo 8
concatenar: nodo 11
ahijar: nodo 7 en columna 32
crear: tipo 0. valor 843
crear: tipo 0. valor -4730
crear: tipo 0. valor 1265
crear: tipo 0. valor -2219
concatenar: nodo 13
crear: tipo 0. valor -4473
crear: tipo 0. valor 2952
imprimir: antes   de colapsar
colapsar: tipo 0
imprimir: despues de colapsar
concatenar: nodo 12
ahijar: nodo 14 en columna 37
crear: tipo 0. valor 3067
crear: tipo 0. valor -3860
crear: tipo 0. valor 751
concatenar: nodo 15
crear: tipo 0. valor 3674
crear: tipo 0. valor -804
crear: tipo 0. valor -2330
concatenar: nodo 23
crear: tipo 0. valor -1710
concatenar: nodo 20
concatenar: nodo 18
crear: tipo 0. valor 2266
crear: tipo 0. valor 3044
ahijar: nodo 25 en columna 27
concatenar: nodo 26
ahijar: nodo 22 en columna 36
crear: tipo 0. valor 3983
concatenar: nodo 21
ahijar: nodo 24 en columna 13
concatenar: nodo 16
crear: tipo 0. valor -783
concatenar: nodo 19
crear: tipo 0. valor 4419
crear: tipo 0. valor 2076
crear: tipo 0. valor -504
crear: tipo 0. valor -1576
imprimir: antes   de colapsar
colapsar: tipo 0
imprimir: despues de colapsar
imprimir: antes   de colapsar
colapsar: tipo 0
imprimir: despues de colapsar
concatenar: nodo 31
concatenar: nodo 29
ahijar: nodo 30 en columna 25
crear: tipo 0. valor 3611
concatenar: nodo 17
crear: tipo 0. valor -648
concatenar: nodo 27
crear: tipo 0. valor -4985
ahijar: nodo 28 en columna 39
crear: tipo 0. valor 437
crear: tipo 0. valor -2664
crear: tipo 0. valor -3288
crear: tipo 0. valor 2366
crear: tipo 0. valor -2068
crear: tipo 0. valor -3171
crear: tipo 0. valor -2253
concatenar: nodo 34
crear: tipo 0. valor -3790
crear: tipo 0. valor -566
crear: tipo 0. valor 4530
crear: tipo 0. valor 3976
crear: tipo 0. valor -4304
crear: tipo 0. valor 347
ahijar: nodo 41 en columna 36
concatenar: nodo 45
crear: tipo 0. valor 2680
ahijar: nodo 47 en columna 44
concatenar: nodo 37
imprimir: antes   de filtrar:
filtrar: tipo 0
imprimir: despues de filtrar:
ahijar: nodo 36 en columna 32
concatenar: nodo 40
concatenar: nodo 49
concatenar: nodo 44
concatenar: nodo 35
concatenar: nodo 48
concatenar: nodo 46
ahijar: nodo 38 en columna 12
concatenar: nodo 39
concatenar: nodo 42
concatenar: nodo 43
imprimir: antes   de colapsar
colapsar: tipo 0
imprimir: despues de colapsar
concatenar: nodo 32
ahijar: nodo 33 en columna 17
borrar
--- test 4 de 50
crear: tipo 1. valor 33.962302
ahijar: nodo 0 en columna 47
crear: tipo 1. valor 73.005634
crear: tipo 1. valor -22.144831
concatenar: nodo 1
ahijar: nodo 2 en columna 10
crear: tipo 1. valor -23.386109
concatenar: nodo 3
crear: tipo 1. valor 37.979852
crear: tipo 1. valor 10.095459
concatenar: nodo 4
concatenar: nodo 5
crear: tipo 1. valor 3.068470
crear: tipo 1. valor 60.091761
concatenar: nodo 6
ahijar: nodo 7 en columna 34
crear: tipo 1. valor -23.728419
crear: tipo 1. valor 16.671376
concatenar: nodo 8
concatenar: nodo 9
imprimir: antes   de colapsar
colapsar: tipo 1
imprimir: despues de colapsar
crear: tipo 1. valor -9.517251
ahijar: nodo 10 en columna 21
crear: tipo 1. valor 17.614542
crear: tipo 1. valor 6.844970
crear: tipo 1. valor -13.655958
crear: tipo 1. valor 23.828514
crear: tipo 1. valor 10.633831
ahijar: nodo 11 en columna 48
concatenar: nodo 14
ahijar: nodo 13 en columna 37
crear: tipo 1. valor 15.687201
ahijar: nodo 12 en columna 34
concatenar: nodo 16
crear: tipo 1. valor 4.131713
concatenar: nodo 17
concatenar: nodo 15
crear: tipo 1. valor 43.613796
crear: tipo 1. valor 61.902360
imprimir: antes   de colapsar
colapsar: tipo 1
imprimir: despues de colapsar
concatenar: nodo 18
imprimir: antes   de filtrar:
filtrar: tipo 1
imprimir: despues de filtrar:
concatenar: nodo 19
imprimir: antes   de colapsar
colapsar: tipo 1
imprimir: despues de colapsar
crear: tipo 1. valor 49.830487
crear: tipo 1. valor 30.545248
concatenar: nodo 21
ahijar: nodo 20 en columna 27
crear: tipo 1. valor -19.689647
crear: tipo 1. valor 40.291228
imprimir: antes   de colapsar
colapsar: tipo 1
imprimir: despues de colapsar
concatenar: nodo 22
ahijar: nodo 23 en columna 8
crear: tipo 1. valor -3.045949
crear: tipo 1. valor 29.300260
crear: tipo 1. valor -4.272554
crear: tipo 1. valor 60.250328
concatenar: nodo 27
crear: tipo 1. valor 21.826462
crear: tipo 1. valor -0.248896
concatenar: nodo 25
crear: tipo 1. valor -7.786972
crear: tipo 1. valor -4.466792
crear: tipo 1. valor 21.423863
imprimir: antes   de filtrar:
filtrar: tipo 1
imprimir: despues de filtrar:
imprimir: antes   de colapsar
colapsar: tipo 1
imprimir: despues de colapsar
crear: tipo 1. valor 57.859116
crear: tipo 1. valor 71.641409
crear: tipo 1. valor -2.987758
concatenar: nodo 26
crear: tipo 1. valor -0.888295
concatenar: nodo 29
imprimir: antes   de colapsar
colapsar: tipo 1
imprimir: despues de colapsar
concatenar: nodo 34
crear: tipo 1. valor -23.673979
crear: tipo 1. valor 8.173616
ahijar: nodo 37 en columna 43
concatenar: nodo 28
concatenar: nodo 33
concatenar: nodo 31
concatenar: nodo 38
crear: tipo 1. valor 14.541209
concatenar: nodo 35
concatenar: nodo 36
concatenar: nodo 39
crear: tipo 1. valor 53.122687
concatenar: nodo 30
imprimir: antes   de filtrar:
filtrar: tipo 1
imprimir: despues de filtrar:
crear: tipo 1. valor -7.672800
ahijar: nodo 24 en columna 34
crear: tipo 1. valor 68.935573
concatenar: nodo 42
crear: tipo 1. valor 31.044704
concatenar: nodo 32
concatenar: nodo 43
ahijar: nodo 41 en columna 6
crear: tipo 1. valor 47.990604
ahijar: nodo 44 en columna 18
crear: tipo 1. valor 13.288099
crear: tipo 1. valor -24.402214
concatenar: nodo 46
imprimir: antes   de colapsar
colapsar: tipo 1
imprimir: despues de colapsar
concatenar: nodo 40
concatenar: nodo 45
crear: tipo 1. valor -0.013715
concatenar: nodo 47
crear: tipo 1. valor 24.351458
ahijar: nodo 48 en columna 2
crear: tipo 1. valor 45.779211
ahijar: nodo 49 en columna 9
borrar
