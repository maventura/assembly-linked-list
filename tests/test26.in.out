antes   de filtrar: { [ 0c2i ] } { [ y6lios ] }despues de filtrar: <vacia>antes   de filtrar: { [ 8qaajik6i ] } { [ 80i037bn ][ l ][ x0vewc ] } { [ f7l8ucyi ][ s ] } { [ 77849 ][ am3p9v ] } { [ f8hhj ] } { [ c9y5lcq ] } { [ 4k ] } { [ nxefwlb ] } { [ 55c5 ] } { [ 3 ] } { [ c ] }despues de filtrar: { [ nxefwlb ] } { [ c ] }antes   de colapsar { [ nxefwlb ] } { [ c ] }